pkg/scheduler/factory.go

```diff
@@ -20,6 +20,23 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
```

Comment:
as I mentioned in the other comment, please revert all changes here, keep the code where it is, only remove the flag.